storage/storage_sqlite.go

```diff
@@ -260,6 +263,46 @@ func (s *SQL) Purge(oldest time.Time) (uint64, error) {
 }
 
 /*
+GetBackup creates a backup of the current database in a temp filename,
+and write the persistent db file to the specified writer.
+*/
+func (s *SQL) GetBackup(w io.Writer) (err error) {
+	backupDbName := fmt.Sprintf("./transicator_sqlite_backup_%v_%v", rand.Int(), time.Now().UnixNano())
```

Comment:
I don't think we should just make this up, and I don't think that we should just put this in ".". Instead we should use ioutil.TempDir, which will generate a random name in the current temporary directory for the process.